alfalfa_web/components/App/App.js

```diff
@@ -64,7 +64,7 @@ class App extends React.Component {
           <div className={this.props.classes.root}>
             <AppBar position="static">
               <Toolbar>
-                <Link to={'/'} className={this.props.classes.title} style={{ textDecoration: 'none', color: 'unset' }}> 
+                <Link to={'/'} className={this.props.classes.title} style={{ textDecoration: 'none', color: 'unset' }}>
```

Comment:
The changes to the JS files are just the copyright year and removing any dangling spaces.